### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by OfficialSterben** [community/community#101755] [Other Feature Feedback, Questions, & Ideas]
Can I host a website directly from a GitHub repository?

**Comment by DanielWren48** [comment]
Hi @OfficialSterben, With platforms like Vercel, Netlify or Heroku you can link your GitHub and select your repo directly, which will indeed deploy your website but make sure your repo is public. 

One service I would recommend is [Render](https://render.com/), it provides a free tear and you can link your GitHub account and deploy with a few click.
